data/content/environment_variables.yaml

```
@@ -215,6 +215,21 @@ variables:
     The GitHub deployment payload data as serialized JSON. Only available on builds triggered by a [GitHub Deployment](https://developer.github.com/v3/repos/deployments/).
   modifable: false
   example: "production"
+- name: BUILDKITE_GROUP_ID
+  desc: |
+    The UUID of the [group step](/docs/pipelines/group-step) that the job belongs to. This variable is only available if the job belongs to a group step.
```

A couple of thoughts:

- I think this reads better without "that." 
- What happens if someone requests the variable when the job doesn't belong to a group step? Does it fail, or return a null value? I think it would be better to describe the outcome. Thinking something like:

```suggestion
    The UUID of the [group step](/docs/pipelines/group-step) the job belongs to. Returns `Null` if the job doesn't belong to a group step.
```

What do you think? This applies to the other two values, too.

---

@mbelton-buildkite the "that" comment makes me realise that I've ended the sentence with a preposition, and grammatically it should be "The UUID of the group step to which the job belongs." I'll let you tell me if I should make that change!

Regarding the return value – these are environment variables, so `null` doesn't really apply here. In the case where the job isn't in a group, the environment variables simply won't be set. How about this?

```suggestion
    The UUID of the [group step](/docs/pipelines/group-step) the job belongs to. This variable will not be set if the job doesn't belong to a group step.
```

---

I'll note I copied this sentence from a few other env vars, so if you prefer this, I could go and update those too